include/ck_tile/ops/fmha/kernel/fmha_fwd_kernel.hpp

```diff
@@ -37,6 +37,23 @@ struct FmhaFwdKernel
 
     using VLayout = ck_tile::remove_cvref_t<typename FmhaPipeline::VLayout>;
 
+    using QElementFunction =
+        ck_tile::remove_cvref_t<typename FmhaPipeline::Problem::ElementFunctions::QElementFunction>;
```

Comment:
I think it's `ElementFunctions`'s responsibility to declare member type using non-reference types. So it's unnecessary to write `remove_cvref_t<>` everywhere

Reply:
29a0670744ccb1e915ef3255df12ef867fcb3c9a